manifests/fpm/pool.pp

```diff
@@ -128,6 +130,7 @@
   $listen_owner                            = undef,
   $listen_group                            = undef,
   $listen_mode                             = undef,
+  $listen_acl_users                        = undef,
```

Comment:
can you please add an appropriate datatype? I'm not sure about all allowed values, but I guess `Optional[String[1]]` is a good start?

Reply:
https://www.php.net/manual/en/install.fpm.configuration.php

> When POSIX Access Control Lists are supported you can set them using this option. When set, listen.owner and listen.group are ignored. Value is a comma separated list of user names. Since PHP 5.6.5. 

as the values are comma separated it would maybe make sense to use an array of strings.
if we add acl_users it maybe also  would make sense to add `listen.acl_groups` too